src/test/java/uk/gov/justice/digital/delius/transformers/OffenderTransformerTest.java

```diff
@@ -33,6 +40,90 @@ public void offenderManagerAllocationReasonNullWhenOffenderTransferAbsent() {
 
     }
 
+    @Test
+    public void disabilitiesCopiedWithLatestFirst() {
+        assertThat(idsOf(offenderTransformer.fullOffenderOf(anOffender()
```

Comment:
You don't really need idsOf(...), you can use something like assertThat(offenderTransformer.fullOffenderOf(..)).extracting("disabilityId").containsExactly(...)